libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SyncOptions.java

```diff
@@ -77,24 +84,37 @@ public static SyncOptions optionsForSyncUp(List<String> fieldlist) {
      * @return
      */
     public static SyncOptions optionsForSyncUp(List<String> fieldlist, MergeMode mergeMode) {
-        return new SyncOptions(fieldlist, mergeMode);
+        return new SyncOptions(fieldlist, null, null, mergeMode);
     }
 
+	/**
+	 * @param fieldlist
+	 * @param createFieldlist
+	 * @param updateFieldlist
+	 * @param mergeMode
+	 * @return
+	 */
+	public static SyncOptions optionsForSyncUp(List<String> fieldlist, List<String> createFieldlist, List<String> updateFieldlist, MergeMode mergeMode) {
```

Comment:
Do we need a public API with both fieldList & create/update Fieldlist or are these mutually exclusive.